src/root/index.js

```diff
@@ -10,22 +11,37 @@ import Multiplexer from './Multiplexer';
 
 require('isomorphic-fetch');
 
+const browserNames = {
+  ie: 'Internet Explorer',
+  edge: 'Edge',
+  firefox: 'Firefox',
+  chrome: 'Chrome'
+};
+const unsupportedBrowsers = ['ie', 'edge'];
```

Comment:
This should only be IE and not Edge?

Reply:
https://github.com/IFRCGo/go-frontend/pull/1566/commits/f953f2436c7116616e00271d1b49a862a941b12a moved the handling to [/src/index.js](https://github.com/IFRCGo/go-frontend/blob/develop/src/index.js) only including IE now